msdbook/tests/test_utils.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from msdbook.utils import fit_logit, plot_contour_map
+from statsmodels.base.wrapper import ResultsWrapper
+
+
+# Define commonly used column names as constants
+PREDICTOR_1 = "Predictor1"
+PREDICTOR_2 = "Predictor2"
+INTERACTION = "Interaction"
+
+
+@pytest.fixture
+def sample_data():
+    """Fixture to provide sample data for testing."""
+    np.random.seed(0)  # For reproducibility
```

Comment:
You should hard code the test values so that you can check the values of `result`